src/main/java/nvt/kts/ticketapp/controller/ticket/TicketController.java

```diff
@@ -0,0 +1,50 @@
+package nvt.kts.ticketapp.controller.ticket;
+
+import nvt.kts.ticketapp.domain.dto.ticket.TicketsDTO;
+import nvt.kts.ticketapp.domain.model.ticket.Ticket;
+import nvt.kts.ticketapp.domain.model.user.User;
+import nvt.kts.ticketapp.exception.user.UserNotFound;
+import nvt.kts.ticketapp.service.ticket.TicketService;
+import nvt.kts.ticketapp.service.user.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.security.Principal;
+import java.util.List;
+
+@RestController
+@RequestMapping(value = "/api/tickets")
+public class TicketController {
+
+    @Autowired
+    private TicketService ticketService;
```

Comment:
Bilo bi dobro kad bismo bili konzistentni oko ovog uvezivanja.
Ja radim na drugaciji nacin jer mislim da ce biti lakse testirati i tako su me ucili na praksi.

Napravis konstruktor sa parametrima i prosledis implementaciju servisa i to dodelis ovom polju ticketService, i nema nigde autowired.
Vidi negde kod mene ili Varge

Reply:
Okej ovo cu prepraviti